core/embed/lib/image.c

```diff
@@ -19,13 +19,13 @@
 
 #include <string.h>
 
-#include "blake2s.h"
 #include "ed25519-donna/ed25519.h"
 
 #include "common.h"
 #include "flash.h"
 #include "image.h"
 #include "model.h"
+#include TREZOR_BOARD
```

Comment:
This explicit #include is probably unnecessary (already included in image.h)

Reply:
removed https://github.com/trezor/trezor-firmware/commit/da2b3a551e37a2b097db42a71251cbe41fb3b93d